bigquery/google/cloud/bigquery/job.py

```diff
@@ -1298,6 +1298,84 @@ def query_plan(self):
         plan_entries = self._job_statistics().get('queryPlan', ())
         return [copy.deepcopy(entry) for entry in plan_entries]
 
+    @property
+    def total_bytes_processed(self):
+        """Return total bytes processed from job statistics, if present.
+
+        See:
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#statistics.query.totalBytesProcessed
+
+        :rtype: int or None
+        :returns: total bytes processed by the job, or None if job is not
+                  yet complete.
+        """
+        return self._job_statistics().get('totalBytesProcessed')
+
+    @property
+    def total_bytes_billed(self):
+        """Return total bytes billed from job statistics, if present.
+
+        See:
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#statistics.query.totalBytesBilled
+
+        :rtype: int or None
+        :returns: total bytes processed by the job, or None if job is not
+                  yet complete.
+        """
+        return self._job_statistics().get('totalBytesBilled')
```

Comment:
Ditto. Client lib needs to convert to int because JSON response has it as a string.